Tests/Integration/Gen/KitchenSink/Sources/KitchenSinkClient.swift

```diff
@@ -15,15 +15,13 @@
 import FirebaseDataConnect
 import Foundation
 
-@available(macOS 10.15, iOS 13, tvOS 13, watchOS 6, *)
```

Comment:
Was it intentional to remove these? Shouldn't the Package.swift versions be updated to these in conjunction with this removal?

Reply:
The style script seems to have removed these. They are part of the generated code.

Reply:
Hmm, the style script isn't touching those when I run it ...

Reply:
Sorry, I think they got stripped out when I regenerated the code. The code is not part of the core SDK. Its generated so we can run the integration tests.

Reply:
We have the version gates on the tests and core sdk so the gate on the gen code isn't required.